asm_processor.py

```diff
@@ -609,7 +609,7 @@ def process_line(self, line, output_enc):
             self.text_glabels.append(line.split()[1])
```

Comment:
this needs to handle jlabel as well, or else ST_FUNC markers will be missing which will result in garbage objdump output